vars/sectionInfraBuild.groovy

```diff
@@ -9,12 +9,20 @@ def call(params) {
   def component = params.component ?: null
   def expires = params.expires ?: LocalDate.now().plusDays(30)
   def pcr = params.pipelineCallbacksRunner
+  def businessArea = env.BUSINESS_AREA_TAG
 
   MetricsPublisher metricsPublisher = new MetricsPublisher(this, currentBuild, product, "")
   approvedEnvironmentRepository(environment, metricsPublisher) {
     withSubscription(subscription) {
       pcr.callAround("buildinfra:${environment}") {
         timeoutWithMsg(time: 150, unit: 'MINUTES', action: "buildinfra:${environment}") {
+          // withAksClient adds the cluster name and RG to env vars  -- only used in CFT Sandbox 
```

Comment:
not needed I think
```suggestion
```

Reply:
@timja perhaps not needed but it took me a while to figure out this was needed to set those env vars, is it useful for the future to stay in as a helper or shall I just remove?

Reply:
I wouldn't, you can search them to see where they come from.